apps/workspace-host/src/interface.js

```diff
@@ -628,6 +641,11 @@ async function _pullImage(workspace) {
         }
 
         const toDatabase = false;
+        cache.set(
+          `workspaceProgressInit:${workspace_image}`,
+          progressDetailsInit,
+          1000 * 60 * 60 * 24, // 24 hours
```

Comment:
I think we should cache for longer than 24 hours -- maybe a month?

Reply:
I have updated the cache expiration to 30 days in 2eefe46.